action.yml

```diff
@@ -145,23 +150,27 @@ runs:
         TAG: ${{ inputs.tag }}
         RELEASE_DESCRIPTION: ${{ inputs.release_description }}
         GITHUB_REPOSITORY: ${{ github.repository }}
+        GITHUB_SHA: ${{ github.sha }}
       run: |
-        # Create new release via GitHub API
-        # https://docs.github.com/en/rest/releases/releases#create-a-release
         # Escape double quotes '"' => '\"'
         RELEASE_DESCRIPTION=${RELEASE_DESCRIPTION//\"/\\\"}
-        RESP_CODE=$(curl -w %{http_code} -s -o /dev/null \
+        # Create new release via GitHub API
+        # https://docs.github.com/en/rest/releases/releases#create-a-release
+        # TODO: change __response.json back to /dev/null
+        RESP_CODE=$(curl -w %{http_code} -s -o __response.json \
         -X POST https://api.github.com/repos/$GITHUB_REPOSITORY/releases \
         -H "Accept: application/vnd.github.v3+json" \
         -H "Authorization: token ${{ github.token }}" \
         -d @- << EOF
         {
           "tag_name": "$TAG",
-          "target_commitish": "${{ github.sha }}",
+          "target_commitish": "$GITHUB_SHA",
           "name": "$TAG",
           "body": "$RELEASE_DESCRIPTION",
           "draft": false,
-          "prerelease": false
+          "prerelease": false,
+          "generate_release_notes": ${{ inputs.release_auto_notes }},
+          "make_latest": "legacy"
```

Comment:
Just noting for if we look back at this in the future and wonder why this legacy change was made: 
"legacy specifies that the latest release should be determined based on the release creation date and higher semantic version."